android/app/src/main/java/com/microsoft/codepush/react/CodePushInvalidPackageException.java

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.codepush.react;
+
+public class CodePushInvalidPackageException  extends RuntimeException {
```

Comment:
What do you think about replacing "Package" with "Update"? I'd like to start eradicating the use of the word "Package" in our codebase :)